src/index.tsx

```diff
@@ -3,6 +3,7 @@ import 'polyfills'
 import Swap, { SwapProps } from 'components/Swap'
 import Widget, { WidgetProps } from 'components/Widget'
 
+export * from './components/Logo'
```

Comment:
Can you explicitly export what you intend to instead of using `*`, to match the other exports?

Reply:
🫡